crates/bevy_render/src/camera/camera.rs

```diff
@@ -1054,8 +1054,18 @@ pub fn extract_cameras(
     ) in query.iter()
     {
         if !camera.is_active {
+            commands.entity(render_entity).remove::<(
+                ExtractedCamera,
+                ExtractedView,
+                RenderVisibleEntities,
+                TemporalJitter,
+                RenderLayers,
+                Projection,
+                GpuCulling,
```

Comment:
In my opinion this is going to be extremely difficult to maintain. The moment that a new component that should be listed here for removal is added, it almost for sure won't.

Is this the only solution? Is there some equivalent of "required components" that automatically removes other components, if one is removed?

Another question (just to understand): what happens when a camera that is made inactive is re-activated? Is there already a solution to re-add these component automatically on reactivation?

Reply:
Do we actually need to remove **all** of those components?
Wouldn't removing `ExtractedView` be enough to halt all processing for this camera?

Reply:
@MiniaczQ Removing `ExtractedView` alone wouldn't fix all the problems. In this PR I was focusing on `render_primitives` example and I started from from trying to fix warnings about camera ambiguity from   `sort_cameras` system. This system uses only `ExtractedCamera` component so it would still spam the logs.
Could less components be removed? Probably. That's what I initially implemented although I don't know if the set I proposed was minimal (it was pretty much arbitrary on my side). For me it makes sense to remove all components that the system inserts and it's easier to reason about than look through all the systems and check which components are important and should be removed and which could be left.
But I agree it's not perfect and could be hard to synchronize. I hope that in the future a better more ergonomic and hopefully generic solution will be found. But for 0.15 I think it is good enough.

Reply:
Oof. Agree with all the comments here, but hope this is at least good motivation to take a look at how this *should* work at the beginning of 0.16.